x-pack/plugins/apm/public/store/urlParams.ts

```diff
@@ -148,8 +152,10 @@ export const getUrlParams = createSelector(
 );
 
 export interface IUrlParams {
+  activeTimelineId: string | null;
   end?: string;
   errorGroupId?: string;
+  flyoutDetailTab: string | null;
```

Comment:
Can we make these `undefined` instead of `null` to be consistent with the others?

Reply:
`toString` will always return `string | undefined` - never `null`

Reply:
Yeah that's fine. I thought I saw spanId being set to null in query params so was just trying to stay somewhat consistent but no idea where I saw that now. I think undefined works the same way for qp